azure-pipelines/codesync-msft.yml

```diff
@@ -39,10 +39,6 @@ steps:
         if [[ $branch_head == "202412" ]]; then
           branch_head=202411
         fi
-        if [[ $branch_head == "202503" ]]; then
-          branch_head=202412
```

Comment:
If you only want to stop code sync from 202412 to 202503:
        if [[ $branch_head == "202503" ]]; then
          continue
        fi